src/pages/home.tsx

```diff
@@ -1,20 +1,55 @@
-import React from "react";
+import React, { useEffect } from "react";
+import PropTypes from "prop-types";
 import { Helmet } from "react-helmet";
 import NavigationBar from "../components/Layout/NavigationBar";
 import Footer from "../components/Layout/Footer";
 import MainPageContainer from "../components/MainPageContainer";
+import { connect } from "react-redux";
+import {
+  resetCertificateState,
+  retrieveCertificateByAction,
+  retrieveCertificateByActionFailure
+} from "../reducers/certificate";
 
-export const HomePage = () => (
-  <>
-    <Helmet>
-      <meta property="og:url" content="https://tradetrust.io" />
-      <meta property="og:title" content="TradeTrust - An easy way to check and verify your documents" />
-      <meta property="og:description" content="Add Tradetrust description" />
-      <meta property="og:url" content={`${window.location.origin}`} />
-      <title>TradeTrust - An easy way to check and verify your documents</title>
-    </Helmet>
-    <NavigationBar active="home" />
-    <MainPageContainer />
-    <Footer />
-  </>
-);
+export const HomePage = props => {
+  useEffect(() => {
+    if (props.location.search !== "") {
+      props.resetCertificateState();
+      const action = JSON.parse(window.decodeURI(props.location.search.substring(3)));
```

Comment:
what is `props.location.search.substring(3)` doing ?

Reply:
TT is using react-router, better to use it

https://github.com/TradeTrust/tradetrust-website/blob/master/package.json#L58